### DISCUSSION TITLE
Censorship

### DISCUSSION THREAD

**Discussion by textbrowser** [textbrowser/biblioteq#237] [Show and tell]
https://www.ala.org/advocacy/fight-censorship?utm_source=ala&utm_medium=topbanner&utm_campaign=redesign

Cataloging applications must offer local digital repositories where physical copies have been removed or are not available. Local repositories also provide means of exchanging digital copies with people.

BiblioteQ provides support for digital items!

**Comment by meteos77** [comment]
Can you please expand in simple words the link you provided because from France, it is difficult to understand the subject.
In any case, BiblioteQ helps freedom and it's a great positive point for all. Thanks again for your investment.

**Reply by textbrowser** [reply]
> Can you please expand in simple words the link you provided because from France, it is difficult to understand the subject. In any case, BiblioteQ helps freedom and it's a great positive point for all. Thanks again for your investment.

I spent a little time reviewing Evergreen and Koha for digital content information. I don't know if these packages include such offerings. It's important that they do and if they do document these features.

**Comment by textbrowser** [comment]
I'd find other censorship efforts in other countries. The link is from the American Library Association so it's an American perspective.
